corehq/apps/app_manager/urls.py

```diff
@@ -12,6 +13,9 @@
     url(r'^languages/bulk_app_translations/upload/$', 'upload_bulk_app_translations', name='upload_bulk_app_translations'),
     url(r'^multimedia/$', 'view_app', name='app_multimedia'),
     url(r'^multimedia_ajax/$', 'multimedia_ajax', name='app_multimedia_ajax'),
+    url(r'^multimedia_ajax_download/$', 'multimedia_ajax_download', name='app_multimedia_ajax_download'),
+    url(r'^multimedia_ajax_upload/$', BulkUploadMultimediaView.as_view(), name='app_multimedia_ajax_upload'),
+    url(r'^multimedia_ajax_references/$', MultimediaReferencesView.as_view(), name='app_multimedia_ajax_references'),
```

Comment:
This is great!

Feel free to ignore if scope creep - but what do you think of getting rid
of the "(Opens in a new window)" next to the mm reference checker button?
(The text looks out of place and I haven't seen that as an indicator
anywhere else on HQ/the internet)

On Tue, Feb 16, 2016 at 12:27 PM, dimagimon <notifications@github.com>
wrote:

> In corehq/apps/app_manager/urls.py
> <https://github.com/dimagi/commcare-hq/pull/10319#discussion_r53045445>:
>
> > @@ -12,6 +13,9 @@
> >      url(r'^languages/bulk_app_translations/upload/$', 'upload_bulk_app_translations', name='upload_bulk_app_translations'),
> >      url(r'^multimedia/$', 'view_app', name='app_multimedia'),
> >      url(r'^multimedia_ajax/$', 'multimedia_ajax', name='app_multimedia_ajax'),
> > +    url(r'^multimedia_ajax_download/$', 'multimedia_ajax_download', name='app_multimedia_ajax_download'),
> > +    url(r'^multimedia_ajax_upload/$', BulkUploadMultimediaView.as_view(), name='app_multimedia_ajax_upload'),
> > +    url(r'^multimedia_ajax_references/$', MultimediaReferencesView.as_view(), name='app_multimedia_ajax_references'),
>
> E501 line too long (117 > 115 characters)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/10319/files#r53045445>.
>